code/modules/unit_tests/security_levels.dm

```diff
@@ -0,0 +1,17 @@
+/**
+ * Security Level Unit Test
+ *
+ * This test is here to ensure there are no security levels with the same name or number level. Having the same name or number level will cause problems.
+ */
+/datum/unit_test/security_levels
+
+/datum/unit_test/security_levels/Run()
+	var/list/comparison_one = subtypesof(/datum/security_level)
+	var/list/comparison_two = LAZYCOPY(comparison_one)
```

Comment:
Why are you copying this at all? You don't mutate `comparison_one`. Just use it directly.

Reply:
I think I planned on doing sometbing with it, I'll change it up